woocommerce/woocommerce
# Update monorepo to pnpm 8

### Submission Review Guidelines:

### Changes proposed in this Pull Request:

Upgrade the monorepo to pnpm 8.



### How to test the changes in this Pull Request:

1. CI should pass
2. You could also test this locally. Update to pnpm 8: `pnpm i -g pnpm`. (double check it worked with `pnpm -v`, then `pnpm i && pnpm build`
